index.html

```diff
@@ -38,5 +38,130 @@ <h4>LET'S CONNECT</h4>
             <li><a href="#"><img class="social" src="./Images/Peace.png" alt="peace"></a></li>
         </ul>
     </section>
+    <!-- second page -->
+    <section class="Works">
+        <!-- card 1 -->
+        <div class="card">
+            <img class="image" src="./Images/Snapshoot-page-2.png" alt="tonic">
+            <h2>Tonic</h2>
+            <ul class="triple">
+                <li class="canopy">CANOPY</li>
+                <li class="double">Back End Dev</li>
+                <li class="double" >2015</li>
+            </ul>
+            <p class="paragraph">
+                A daily selection of privately personalised reads; no accounts or no sign-ups required.
+            </p>
+            <div class="langs">
```

Comment:
- [ ] Kindly please add a `<ul>` tag here instead of a `<div>`.  Inside that element, your list items let's make them `<li>` each. That way, the code will be a lot more semantic, and the browsers will have an easier time rendering it. 💪